src/mic/_menu.py

```diff
@@ -282,7 +292,10 @@ def call_menu_select_property(mapping, resource_object, full_request=None, paren
     while True:
         click.clear()
         first_line_new(resource_object.name)
-        property_chosen = menu_select_property(request, mapping)
+        if resource_object.name=='Model'or resource_object.name=='Model Configuration':
```

Comment:
Nope. This is a bad idea, you must verify it in a generic way.

Reply:
What could be a more generic way? Verify if it's a top resource?
I think this is problematic because if you are defining a model and define versions and then configurations, you don't want to allow inserting them from there.

Reply:
So, what should be the generic way. I thought only model and configurations are allowed to have send. 
Should I have array similar to ACTIONS declared on top to denote which can have send option?

Reply:
See my comment. If ModelConfigurations can send, then you can encounter an error when defining a model configuration on a model version of a model. I think that if what you are editing is a top resource, then you can send.

Reply:
So, only model can have send option, is that correct ?

Reply:
No. If you start with a modelConfiguration, then modelConfig is your top resource and you should be able to send. If you start with model, then you should not be able to send in the sub-resources